src/styles/block/cards.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  height: 406px;
+  border: 1px solid $borderInCard;
+  border-radius: 5px;
+  transition: $transition-all;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    margin: 32px 19px 0;
+  }
+
+  &__photo {
+    display: block;
+    max-width: 100%;
+    height: 134px;
+    // height: auto;
```

Comment:
Remove comments

Reply:
still not fixed
<img width="797" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/70220667/1ed6513b-d303-4119-b5dc-6dd1362e84fc">